gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManager.java

```diff
@@ -138,20 +141,21 @@ public void handleMinioEvent(NotificationRecords notificationRecords) {
 
     public void updateTasks(Event event) {
         TaskManagerConfigurationProperties.ProcessProperties processProperties = taskManagerConfigurationProperties.getProcess();
-        if (!event.userMetadata().isEmpty() && processProperties.getTag().equals(event.userMetadata().get(FILE_PROCESS_TAG))
-                && processProperties.getInputs().contains(event.userMetadata().get(FILE_TYPE))) {
-            String fileType = event.userMetadata().get(FILE_TYPE);
-            String validityInterval = event.userMetadata().get(FILE_VALIDITY_INTERVAL);
+        if (!event.userMetadata().isEmpty() && processProperties.getTag().equals(event.userMetadata().get(FILE_TARGET_PROCESS_METADATA_KEY))
+                && processProperties.getInputs().contains(event.userMetadata().get(FILE_TYPE_METADATA_KEY))) {
```

Comment:
It takes into account only files that are part of input list not output list. Maybe we can just filter on target process here.